src/utilities/downloads_backend.jl

```diff
@@ -12,8 +12,7 @@ which is the `Downloads.Downloader` to use. If set to `nothing`, the default,
 then a global downloader object will be used.
 
 Downloads.jl tends to perform better under concurrent operation than HTTP.jl,
-particularly with `@async` / `asyncmap`. As of March 2022, threading (e.g. `@spawn` or `@threads`) with Downloads.jl is broken on all releases of Julia ([Downloads.jl#110](https://github.com/JuliaLang/Downloads.jl/issues/110)), and there are still reported issues on the upcoming
-1.7.3 and 1.8 releases ([Downloads.jl#182](https://github.com/JuliaLang/Downloads.jl/issues/182])).
+particularly with `@async` / `asyncmap`.
```

Comment:
```suggestion
particularly with `@async` / `asyncmap`. Note that threading (e.g. `@spawn` or `@threads`) with Downloads.jl is broken on Julia releases prior to 1.8 (https://github.com/JuliaLang/Downloads.jl/issues/182#issuecomment-1069269944).
```

Seems like it's not threadsafe on anything prior to 1.8 according to that comment by @StefanKarpinski (though couldn't ascertain 100% from looking over the linked issues, e.g. whether any fix finally made it to 1.7 or even 1.6?).

It would be nice if Downloads.jl mentioned in its README on which Julia versions it is threadsafe (bonus linking to issues).